voctogui/lib/videodisplay.py

```diff
@@ -24,6 +25,13 @@ def __init__(self, drawing_area, port, width=None, height=None,
         use_previews = Config.getboolean('previews', 'enabled') \
             and Config.getboolean('previews', 'use')
 
+        audiostreams = int(Config.get('mix', 'audiostreams'))
+
+        if (Config.has_option('mainvideo', 'vumeter')) \
```

Comment:
Same as above, `get` will fail if the Option is missing, so `has_option` is unneccessary